android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -632,4 +696,60 @@ public void setPlayInBackground(boolean playInBackground) {
     public void setDisableFocus(boolean disableFocus) {
         this.disableFocus = disableFocus;
     }
+    
+    /**
+     *
+     * @param drmName
+     */
+    public void setDrmName(String drmName) throws ParserException{
+        switch (Util.toLowerInvariant(drmName)) {
+            case "widevine":
+                this.drmUUID = C.WIDEVINE_UUID;
+                break;
+            case "playready":
+                this.drmUUID = C.PLAYREADY_UUID;
+                break;
+            case "cenc":
```

Comment:
I am not a DRM expert but AFAIK cenc != clearkey.
Clearkey is a protection system like Widevine / PlayReady / Verimatrix / etc

And CENC is common encryption, and is an encryption format. AFAIK Clearkey / PlayReady / Widevine can be encrypted in CENC

https://www.html5rocks.com/en/tutorials/eme/basics/#common-encryption